File: go/circle-of-numbers/circle-of-numbers.go
package main

import(
	"fmt"
)

func circleOfNumbers(n, firstNumber int)int{
	return (n / 2 + firstNumber) % n
}


func main(){
	n, firstNumber := 10 , 2

	fmt.Printf("The radially opposite position to %d in a circle starting with 0 and ending with %d is: %d ",firstNumber, n-1, circleOfNumbers(n, firstNumber))
}

File: js/circle-of-numbers.js
function circleOfNumbers(n ,firstNumber){
    return (n / 2 + firstNumber) % n
}

let n = 10, firstNumber = 2

console.log(`The radially opposite position to ${firstNumber} in a circle starting with 0 and ending with ${n-1} is:`, circleOfNumbers(n, firstNumber))